examples/freetype_gl/single_character/src/main_loop.adb

```diff
@@ -43,21 +39,19 @@ procedure Main_Loop (Main_Window : in out Glfw.Windows.Window) is
    Colour_Location       : GL.Uniforms.Uniform;
    Projection_Location   : GL.Uniforms.Uniform;
    Texture_Location      : GL.Uniforms.Uniform;
-   Position_Location     : GL.Attributes.Attribute;
    Projection_Matrix     : Singles.Matrix4;
 
-   Back_Colour   : constant GL.Types.Colors.Color := (0.3, 0.6, 0.6, 1.0);
+   Back_Colour    : constant GL.Types.Colors.Color := (0.3, 0.6, 0.6, 1.0);
+   Main_Exception : Exception;
```

Comment:
Why is this necessary? Declaring an exception in a package body is very bad style, as no-one can catch it from outside. There should be a better way to fix compiler warnings than introducing this exception.